src/app/App.tsx

```diff
@@ -19,50 +19,62 @@ import dialogueBoxPlugin from '../plugins/dialogue_box/DialogueBox';
 import AccountMenu from './components/accountMenu';
 
 function App() {
-  const [open, setOpen] = useState<boolean>(false);
-
   return (
     <Router>
-      <div>
-        <header className="layout_header">
-          <BurgerMenu open={open} setOpen={setOpen}>
-            <Link to="/users">All Users</Link>
-            <Link to="/projects">All Projects</Link>
-          </BurgerMenu>
-          <AccountMenu></AccountMenu>
-        </header>
+      <Switch>
+        <Route path="/viewer/:id">
+          <Viewer />
+        </Route>
+        <Route path="/">
+          <AppWithBurger />
+        </Route>
+      </Switch>
+    </Router>
+  );
+}
 
-        <Switch>
-          <Route path="/login">
-            <Login />
-          </Route>
+function AppWithBurger() {
```

Comment:
Well, we may call that the full mode, full app or similar names. Not just AppWithBurger? We probably can also comment here why we have these two functions